packages/fxa-content-server/app/scripts/templates/pair/index.mustache

```diff
@@ -6,17 +6,17 @@
   <section>
     <div class="error"></div>
 
-    <div class="graphic {{ graphicId }}">Ready the code scanner on your Firefox app.</div>
-    <p>Ready the code scanner on your Firefox app.</p>
+    <div class="graphic {{ graphicId }}">{{#t}}Open the code scanner on your Firefox app and show code when ready.{{/t}}</div>
```

Comment:
Seems a bit strange to have this text duplicated.

Reply:
gonna remove it